python_benchmarks/conftest.py

```diff
@@ -28,3 +29,8 @@ def disable_benchmarking(request):
 
 def pytest_make_parametrize_id(val):
     return repr(val)
+
+
+def pytest_benchmark_update_machine_info(config, machine_info):
+    for property, value in DEVICE_PROPERTIES.items():
+        machine_info[property] = value
```

Comment:
Is this the same as `machine_info.update(DEVICE_PROPERTIES)`?

Reply:
Yes, done.